brayns/Brayns.cpp

```diff
@@ -521,6 +528,21 @@ struct Brayns::Impl
 
 #if (BRAYNS_USE_BRION)
     /**
+        Loads data from a neuron matrix file (command line parameter
+       --matrix-file)
```

Comment:
--connectivity-file rather than matrix? Sounds too generic imo. In the end, everything is a matrix :)